src/stylesheets/components/_buttons.scss

```diff
@@ -9,111 +9,40 @@ button {
   @include btn;
 }
 
-/*--- Default Button Toggle ---*/
-
-.btn--toggle {
-  border: 0;
-  display: none;
-  position: relative;
-  background-color: transparent;
-  padding-right: $base-spacing-unit-small;
-
-  &::after {
-    content: '';
-    display: inline-block;
-    position: absolute;
-    top: 50%;
-    right: 0;
-    vertical-align: middle;
-    border: $base-spacing-unit solid transparent;
-    border-bottom-width: 0;
-    border-top-color: $grey-1;
-    margin: -($base-spacing-unit / 2) 0 0 $base-spacing-unit-tiny;
-  }
-}
-
-.has-js .btn--toggle {
-  display: inline;
-}
-
-/*--- Button Toggle [within the header] ---*/
-
-header .btn--toggle {
-  top: $base-spacing-unit;
-  right: 0;
-  width: 57px;
-  height: 50px;
-  position: absolute;
-  background-color: $deep-purple;
-  padding: $base-spacing-unit $base-spacing-unit-small;
-  @media (max-width: $medium - 1) {
-    display: inline;
-  }
-
-  &:after {
-    display: none;
-  }
-  &:hover, &:focus, &:active {
-    outline: none;
-
-    svg[data-icon="search"] g {
-      fill: $grey-3;
-    }
-    svg[data-icon="close"] g {
-      stroke: $grey-3;
-    }
-  }
-}
-
-.has-js header .btn--toggle {
-  @include media($medium) {
-    display: none;
-  }
-}
-
-/*--- Button Toggle within components ---*/
+/*--- Dropdown buttons ---*/
 
 .dropdown,
 .dropdown--block {
-  .btn--toggle {
+  > button {
     margin: 0;
   }
-}
-
-.dropdown.open,
-.dropdown--block.open {
-  .btn--toggle::after {
+  > button::after {
+    content: '';
+  }
+  > button:not([class*="btn"]) {
+    border: 0;
+    background-color: transparent;
+    @include btn--toggle($grey-1, true);
+    @include links($grey-1, none, $grey-1, $grey-1);
+    @include typography--heavy;
+  }
+  &.open > button::after {
     transform: rotate(180deg);
   }
 }
 
-.dropdown--block .btn--toggle {
-  border: 0;
+.dropdown > button:not([class*="btn"]) {
+  padding-left: 0;
+}
+
+.dropdown--block > button {
   width: 100%;
-  display: inline;
-  text-align: left;
+  background-color: transparent;
   padding: $base-spacing-unit-small;
-  @include typography--heavy;
-  @include links($grey-1, none, $grey-1, $grey-1);
-
-  &:hover, &:focus {
-    text-decoration: underline;
-    background-color: transparent;
-    outline: none;
-  }
-  &::after {
-    position: relative;
-  }
 }
 
-[class*="block"] .dropdown .btn--toggle {
-  padding: 0 $base-spacing-unit-large 0 0;
-  @include typography--heavy;
-  @include links($grey-1, none, $grey-1, $grey-1);
-
-  &:hover, &:focus {
-    text-decoration: underline;
-  }
+body:not(.has-js) .dropdown > [class*="btn"] {
```

Comment:
Can this instead by `body.has-js .dropdown > button` { display: block }

Reply:
This is done. However, now it's hiding any toggle button directly descendant to dropdowns. Before it was hiding buttons that look like buttons rather than text. I think this approach makes more sense because those buttons become un-clickable when JS is disabled, but might be a problem for dropdowns that say, "When to contact an MP".